### PR TITLE
Feature/improve demo app

### PR DESCRIPTION
**Pull request by SungjunApp**
This is only to improve the UI of our demo app.

What's changed
- improved the design of grid and linear list
- Improved the video player and the image viewer and added a product list at the bottom
- added an animated Pixlee loading icon. (the look is made by a JSON file) I used Lottie library to do this
    - Lottie: (https://lottiefiles.com/)
         - available: Android, iOS, Web, React Native, and Windows

### CODE REVIEW

File: app/src/main/java/com/pixlee/pixleeandroidsdk/ui/gallery/GalleryFragment.java [hunk 1]
```diff
@@ -146,13 +155,35 @@ private void loadAlbum() {
         PXLAlbumSortOptions so = new PXLAlbumSortOptions();
         so.sortType = PXLAlbumSortType.APPROVED_TIME;
         so.descending = true;
-        album.setPerPage(20);
+        //album.setPerPage(9);
```
Comment (jchen123) [inline app/src/main/java/com/pixlee/pixleeandroidsdk/ui/gallery/GalleryFragment.java:158]: we dont need per page?

File: app/src/main/java/com/pixlee/pixleeandroidsdk/ui/gallery/GalleryFragment.java [hunk 2]
```diff
@@ -248,19 +289,21 @@ public void onError(String error) {
...
     @Override
     public void onComplete(ArrayList<PXLPhoto> photos) {
+        setLoading(false);
         if (photos == null) {
             return;
         }
-        album.openedWidget(PXLWidgetType.horizontal);
-        album.openedWidget("gallery");
-        album.loadMore();
+        //album.openedWidget(PXLWidgetType.horizontal);
```
Comment (jchen123) [inline app/src/main/java/com/pixlee/pixleeandroidsdk/ui/gallery/GalleryFragment.java:296]: there are a lot of commented code in this PR in general. can we have explanations for the commented code or just deletion of the commented code? theres no point in just commenting out code

File: app/src/main/java/com/pixlee/pixleeandroidsdk/ui/gallery/GalleryFragment.java (hunk 1 above)
Comment (SungjunApp) [reply app/src/main/java/com/pixlee/pixleeandroidsdk/ui/gallery/GalleryFragment.java:158]: The API of Distillery returns 20 items as a default. I just removed it to use the default. But, there is no example of album.setPerPage(..). I'll restore the code.

File: app/src/main/java/com/pixlee/pixleeandroidsdk/ui/gallery/GalleryFragment.java (hunk 2 above)
Comment (SungjunApp) [reply app/src/main/java/com/pixlee/pixleeandroidsdk/ui/gallery/GalleryFragment.java:296]: Agree! I'll add GUI views for developers to try these features.

Comment (SungjunApp) [reply app/src/main/java/com/pixlee/pixleeandroidsdk/ui/gallery/GalleryFragment.java:438]: I implemented all sort options and half of the filter options for the SDK users to understand about it and left more explanations for the other half of the filter options.

**Comment by SungjunApp** [comment]
@kj13ennett @jchen123 Please guys review this. Most changes are UI related.

**Pull request merged**